lib/stdlib/src/binary.erl

```diff
@@ -458,15 +541,16 @@ the following example:
 
 ```erlang
 1> binary:matches(<<"abcde">>,
-                  [<<"bcde">>,<<"bc">>,<<"de">>],[]).
+                  [<<"bcde">>,<<"bc">>,<<"de">>],
+                  []).
 [{1,4}]
 ```
 
 The result shows that <<"bcde">> is selected instead of the shorter match
-<<"bc">> (which would have given raise to one more match, <<"de">>). This
-corresponds to the behavior of POSIX regular expressions (and programs like
-awk), but is not consistent with alternative matches in `re` (and Perl), where
-instead lexical ordering in the search pattern selects which string matches.
+<<"bc">> (which would have given rise to one more match, <<"de">>). This
```

Comment:
```suggestion
<<"bc">> (which would have given one more match, <<"de">>). This
```

Reply:
I think a prefer: "which would have resulted in one more match"

Reply:
Works for me, I was just unhappy about about the "rise to" part.